python/perspective/perspective/table/_accessor.py

```diff
@@ -104,6 +96,29 @@ def __init__(self, data_or_schema):
 
         self._types = []
 
+        # Verify that column names are strings, and that numpy arrays are of type `ndarray`
+        if six.PY2:
+            valid_name_types = (str, unicode)  # noqa: F821
```

Comment:
`isinstance(whatever, six.string_types)`

Reply:
just keep this bookmarked for any time youre not sure https://six.readthedocs.io/#module-six